src/shippingModeSelection/shipping-mode-selection.html

```diff
@@ -46,9 +46,7 @@
         <!-- MONDIAL RELAY CHOICE -->
         <div class="animate-mondial-relay mb-3"
              data-ng-if="$ctrl.selectedMode === 'mondialRelay'">
-            <div data-mondial-relay
-                 data-ng-model="$ctrl.selectedRelay">
-            </div>
+            <mondial-relay data-ng-model="$ctrl.selectedRelay"></mondial-relay>
```

Comment:
In addition you can specify the missing peer dependency to `@ovh-ux/ng-ovh-mondial-relay` by simply running:

```sh
yarn add --peer @ovh-ux/ng-ovh-mondial-relay
```